README.md

```diff
@@ -34,3 +34,35 @@ python3 ./mower.py --address D8:B6:73:40:07:37
 ```
 
 You can uncomment parts of `async def main(mower)` to send commands
+
+## Debugging on an android phone
+
+You can get Bluetooth debug logs from an android phone which will help for unknown codes
+or implementation of extra devices. To debug you need to enable developer mode on your
+android handset/tablet (and have the manufacturer app installed to communicate with your mower)
+
+To enable debug mode on android:
+```shell
```

Comment:
This isn't a shell command, so it shouldn't be inside the shell code block. I don't think it should be in a code block at atll

Reply:
Understood. please see updated commit